src/closedWindows/closedCard-directive.js

```diff
@@ -0,0 +1,14 @@
+(function() {
+    'use strict';
+
+    angular.module('openfin.closedCard')
+        .directive('closedCard', [() => {
+            return {
+                restrict: 'E',
+                templateUrl: 'closedWindows/closedCard.html',
+                scope: {
+                    cwindow: '='
```

Comment:
I think I agree, I wasn't sure what `cwindow` was for a while (in fact it's not a window either, but instead a JSON object representing the window state)